protoc-gen-swagger/options/openapiv2.proto

```diff
@@ -0,0 +1,223 @@
+syntax = "proto3";
```

Comment:
Nice touch on reserving all the numbers for the order they appear in the swagger spec!